tests/New-DbaAgentOperator.Tests.ps1

```diff
@@ -0,0 +1,27 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
+    Context "Validate parameters" {
+        [object[]]$params = (Get-Command $CommandName).Parameters.Keys | Where-Object { $_ -notin ('whatif', 'confirm') }
```

Comment:
We've changed this pattern to the following format if you want to update it. The test is failing because your `$knownParameters is missing one that is expected (or has one extra).

https://github.com/sqlcollaborative/dbatools/blob/05fbf9898caf3922d1c9bf8017465bd57e5d387a/tests/Install-DbaFirstResponderKit.Tests.ps1#L6-L13